app/components/HowCanIHelp/HelpItem.tsx

```diff
@@ -0,0 +1,22 @@
+import {ReactNode} from 'react'
+import LinkCard, {Link} from '~/components/LinkCard'
+
+type HelpItemProps = {
+  title?: ReactNode
+  children: ReactNode
+  links?: Link[]
+}
+const HelpItem = ({title, children, links}: HelpItemProps) => {
```

Comment:
this name is bad. It's basically each one of these thingies:
![image](https://github.com/user-attachments/assets/6f6fa066-c93e-4072-94de-c41614eb3a27)

Reply:
Nice